.github/workflows/publish_docs.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Documentation
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  linux:
+    runs-on: ubuntu-18.04
+    env:
+      IREE_DOC_BUILD_DIR: build-docs
+    steps:
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_WRITE_ACCESS_TOKEN }}
+      - name: Fetching gh-pages branch
+        run: |
+          git fetch origin gh-pages:refs/remotes/origin/gh-pages
```

Comment:
Isn't that the default fetch location? Seems like you should be able to just `git fetch origin gh-pages`

Reply:
Yeah, indeed. Fixed.